.github/workflows/demo-server-down.yml

```diff
@@ -6,33 +6,46 @@ on:
     paths-ignore:
       - '**.md'
       - 'docs/**'
+  workflow_dispatch:
 
 env:
   B2C_APP_SUFFIX: universal-b2c
   B2B_APP_SUFFIX: universal-b2b
 
 jobs:
   RemoveDemoServer:
-    if: github.event.pull_request.head.repo.owner.login == 'intershop'
+    if: github.event.repository.full_name == 'intershop/intershop-pwa'
     runs-on: ubuntu-latest
     steps:
+      - name: Determine App Names
+        run: |
+          if [[ -n ${{ github.event.pull_request.head.ref }} ]]
+          then
+            REF=${{ github.event.pull_request.head.ref }}
+          else
+            REF=${{ github.event.ref  }}
+          fi
+          APP_NAME=$(printf '%.60s' "${{ secrets.AZURE_DEMO_RESOURCEGROUP }}-$REF" | sed -e 's/refs\/heads\///' | sed -e 's/\//-/g')
+          echo "APP_NAME=$APP_NAME" >> $GITHUB_ENV
+          echo "APP_NAME_B2C=$APP_NAME-${{ env.B2C_APP_SUFFIX }}" >> $GITHUB_ENV
```

Comment:
Is that still necessary? The two suffixes for b2b and b2c are leftovers from the time we deployed two appservices. Today we only have one.